src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,39 @@
+import classNames from "classnames";
+import { NavLink } from "react-router-dom";
+
+export const Navigation = () => {
+  return (
+    <nav
+      data-cy="nav"
+      className="navbar is-fixed-top has-shadow"
+      role="navigation"
+      aria-label="main navigation"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink
+            to="/"
+            className={({ isActive }) =>
+              classNames("navbar-item", {
+                "has-background-grey-lighter": isActive,
+              })
+            }
+          >
+            Home
+          </NavLink>
+
+          <NavLink
+            to="/people"
+            className={({ isActive }) =>
+              classNames("navbar-item", {
+                "has-background-grey-lighter": isActive,
+              })
+            }
+          >
+            People
+          </NavLink>
```

Comment:
They code looks the same what about creating object with items and then render NavLink's using map. Remember DRY.